aiplatform/src/main/java/aiplatform/batchpredict/CreateBatchPredictionGeminiBigqueryJobSample.java

```diff
@@ -0,0 +1,108 @@
+    String bigquerySourceInputUri =
+        "bq://storage-samples.generative_ai.batch_requests_for_multimodal_input";
+    String predictionsFormat = "bigquery";
+    String bigqueryDestinationOutputUri = "bq://PROJECT_ID.MY_DATASET.MY_TABLE";
+
+    createBatchPredictionGeminiBigqueryJobSample(
+        project,
+        displayName,
+        modelName,
+        instancesFormat,
+        bigquerySourceInputUri,
+        predictionsFormat,
+        bigqueryDestinationOutputUri);
+  }
+
+  public static void createBatchPredictionGeminiBigqueryJobSample(
+      String project,
+      String displayName,
+      String model,
+      String instancesFormat,
+      String bigquerySourceInputUri,
+      String predictionsFormat,
+      String bigqueryDestinationOutputUri)
+      throws IOException {
+    JobServiceSettings settings =
+        JobServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (JobServiceClient client = JobServiceClient.create(settings)) {
+      // Passing in an empty Value object for model parameters;
+      // Add model parameters per request in the input BigQuery table.
+      Value modelParameters = ValueConverter.EMPTY_VALUE;
+
+      BigQuerySource bigquerySource =
+          BigQuerySource.newBuilder().setInputUri(bigquerySourceInputUri).build();
+      BatchPredictionJob.InputConfig inputConfig =
+          BatchPredictionJob.InputConfig.newBuilder()
+              .setInstancesFormat(instancesFormat)
+              .setBigquerySource(bigquerySource)
+              .build();
+      BigQueryDestination bigqueryDestination =
+          BigQueryDestination.newBuilder().setOutputUri(bigqueryDestinationOutputUri).build();
+      BatchPredictionJob.OutputConfig outputConfig =
+          BatchPredictionJob.OutputConfig.newBuilder()
+              .setPredictionsFormat(predictionsFormat)
+              .setBigqueryDestination(bigqueryDestination)
+              .build();
+      String modelName =
+          String.format(
+              "projects/%s/locations/%s/publishers/google/models/%s", project, location, model);
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              .setModel(modelName)
+              .setModelParameters(modelParameters)
+              .setInputConfig(inputConfig)
+              .setOutputConfig(outputConfig)
+              .build();
+      LocationName parent = LocationName.of(project, location);
+      BatchPredictionJob response = client.createBatchPredictionJob(parent, batchPredictionJob);
+      System.out.format("response: %s\n", response);
+      System.out.format("\tName: %s\n", response.getName());
```

Comment:
Kindly opt to return the value instead of sysout.

Reply:
Done